src/_data/LiveData.js

```diff
@@ -81,7 +81,7 @@ const initAuthorized = (authData, store) => {
     api.subscribeToAllOpenContracts();
     subscribeToWatchlist(store);
 
-    const isVirtual = authData.authorize.loginid.startsWith('VRTC');
+    const isVirtual = authData.authorize.is_virtual === '1';
```

Comment:
Not sure where should I place this, how about let it stay here until we actually find we need them more?